Description: The embedded-repo.jar is not built. To avoid a FTBFS the error is commented out.
Author: Markus Koschany <apo@debian.org>
Forwarded: not-needed
--- a/biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java
+++ b/biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java
@@ -490,7 +490,6 @@
 								}
 							}
 						}
-						error("Could not find " + EMBEDDED_REPO + " as a resource on the classpath");
 					}
 					return false;
 				} finally {
--- a/biz.aQute.bnd.gradle/bnd.bnd
+++ b/biz.aQute.bnd.gradle/bnd.bnd
@@ -9,8 +9,7 @@
 	aQute.libg;version=project, \
 	biz.aQute.bndlib;version=latest, \
 	biz.aQute.resolve;version=latest, \
-	biz.aQute.repository;version=latest, \
-	biz.aQute.bnd.embedded-repo;version=latest
+	biz.aQute.repository;version=latest
 
 -testpath: \
 	${junit}
